build/helpers/moment.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var moment = require('moment');
+
+module.exports = function (context, options) {
+	var date = options ? context : new Date();
+	var options = options || context;
+	var format = options.hash.format || 'YYYY-MM-DD ';
```

Comment:
If you call the helper with no value (`context`) and only supply a hash, then `context` will become `options` and `options` will be `undefined`.

```hbs
{{moment format="YYYY"}}
```